S21/pnklshh/refactorme.py

```diff
@@ -1,84 +1,128 @@
 import unicodedata
 
-class Word:
+class Word(object):
+
+  def __init__(self, word, bcpCode):
+    self._word = word
+    self._language = bcpCode
+
+  def toLower(self):
+      return self._word.lower()
+
+class IrishWord(Word):
 
   def __init__(self, word, bcpCode, std=False):
-    self._w = word
-    self._l = bcpCode
-    self._finalSigma = False
+    Word.__init__(self, word, bcpCode)
     self._standardIrishSpelling = std
-    # OLD EXPERIMENTAL CODE for dealing with vowel harmony
-    # self._numVowels = 0
-    # for c in word:
-    #   if c in 'aeiouAEIOU':
-    #   self._numVowels += 1
 
-  def setWord(self, w):
-    self._w = w
+  def specialCase(self):
```

Comment:
Python style: I like a leading underscore in this name since it's only used internally by the class: _specialCase